### ISSUE TITLE
Ability to disable authentication fallback storage warning

### ISSUE THREAD

**Issue by pavelzw** [conda/rattler#422]
In some use cases (for example building with github actions or running pixi in a docker container with no keychain) I get the repeated warning

```
WARN rattler_networking::authentication_storage::fallback_storage: Can't find path for fallback storage on /home/runner/.rattler/rattler_auth_store.json
```

Since in these use cases, it's intended to not have a keychain, I would like to not see this warning. Maybe with `RATTLER_NO_FALLBACK_STORAGE_WARNING=1`?

**Comment by baszalmstra** [comment]
@pavelzw is this still the case?

**Comment by wolfv** [comment]
Yeah it is still the case, but we forcefully disable the logs in `rattler-build` nowadays

**Comment by pavelzw** [comment]
I actually also haven't seen this issue with pixi in docker containers / GHA in a while...
